compliance-by-design-demo/package.json

```diff
@@ -8,15 +8,20 @@
   "dependencies": {
     "@discipl/core-ephemeral": "^0.11.0",
     "@discipl/law-reg": "^0.5.4",
+    "async-redis": "^1.1.7",
```

Comment:
This adds a lot of dependencies to the frontend, can we do this in a separate folder with its own package.json?

Reply:
I was trying to avoid making separate packages, but I would definitely be more clean and efficient. I am going to refactor that.

Reply:
Fixed